### DISCUSSION TITLE
wlll there be migration tool to migration form 30 to 0.31 pipelines ?

### DISCUSSION THREAD

**Discussion by zhihengGet** [open-circle/valibot#604] [General]
i saw this  https://github.com/fabian-hiller/valibot/pull/591 but seems like grit does not support windows natively, and rewriting all my valibot schema would be a huge work :(

**Comment by fabian-hiller** [comment]
There are currently two codemods, but I haven't had time to check if they cover most cases. I will add them to the migration guide if they cover most cases. Feel free to try [this codemod](https://codemod.com/registry/valibot-0-30-0) and let me know if it works for you.
